test/directoryAsModule/b/index.js

```diff
@@ -0,0 +1 @@
+module.exports = 'b'
```

Comment:
Can we add a second file to this directory, to explicitly test (and convey) that it isn't getting `require`d?